blueocean-core-js/src/js/i18n/i18n.js

```diff
@@ -164,7 +164,23 @@ export default function i18nTranslator(pluginName, namespace, onLoad) {
                 logger.log('Translator instance created for "%s". Language detected as "%s".', translatorCacheKey, detectedLang);
             }
 
-            translator = I18n.getFixedT(detectedLang, namespace);
+            const fixedT = I18n.getFixedT(detectedLang, namespace);
+            translator = function (i18nKey, i18nParams) {
+                const normalizedKey = i18nKey.replace(/[\W]/g, '.');
+                let passedParams = i18nParams;
+                if (normalizedKey !== i18nKey) {
+                    if (!passedParams) {
+                        passedParams = {};
+                    }
+                    if (!passedParams.defaultValue) {
+                        passedParams.defaultValue = i18nKey;
+                    }
+                    if (logger.isDebugEnabled()) {
+                        logger.debug(`Normalized i18n key "${i18nKey}" to "${normalizedKey}".`);
+                    }
+                }
+                return fixedT(normalizedKey, passedParams);
+            };
```

Comment:
The above code is about making it a bit easier to just do the translation without having to define defaults etc i.e. just specify the default string and the key and default are inferred from that I think it's a bit better Vs having to define the `defaultValue` which, while not too bad, was a bit of a drag. See af675a1.